usaspending_api/references/tests/test_financial_spending.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import json
+from datetime import date
+
+from model_mommy import mommy
+from rest_framework import status
+
+from usaspending_api.awards.models import Award
+from usaspending_api.references.models import Agency, ToptierAgency, SubtierAgency
+
+
+@pytest.fixture
+def financial_spending_data(db):
+    # agency w/ tt agency
+    ttagency1 = mommy.make('references.ToptierAgency', cgac_code="cgac_code", name="tta_name")
+    mommy.make('references.Agency', id=1, toptier_agency=ttagency1)
+    # TAS w/ same ttagency
+    tas1 = mommy.make('accounts.TreasuryAppropriationAccount', awarding_toptier_agency=ttagency1, funding_toptier_agency=ttagency1)
+
+    # Financial Account with Object class and submission
+    object_class_1 = mommy.make('references.ObjectClass', major_object_class="mocCode",
+                                major_object_class_name="mocName", object_class="ocCode", object_class_name="ocName")
+    submission_1 = mommy.make('submissions.SubmissionAttributes', reporting_fiscal_year=2017, cgac_code="cgac_code")
+    mommy.make('financial_activities.FinancialAccountsByProgramActivityObjectClass', object_class=object_class_1,
+               obligations_incurred_by_program_object_class_cpe=1000, submission=submission_1, treasury_account=tas1,
+               final_of_fy=True)
+
+
+@pytest.mark.django_db
+def test_award_type_endpoint(client, financial_spending_data):
+    """Test the award_type endpoint."""
+
+    resp = client.get('/api/v2/financial_spending/object_class/?fiscal_year=2017&funding_agency_id=1')
+    print(resp.data)
+    assert resp.status_code == status.HTTP_200_OK
+    assert len(resp.data['results']) > 0
```

Comment:
I know that setting up the test models here is tedious (having done it for the `financial_balances/agencies/` endpoint). Since you've gotten this far, I think it would be worthwhile to keep going and:

1. explicitly test for the number of result you're expecting (instead of `> 0`)
2. add a `FinancialAccountsByProgramActivityObjectClass` object with a different tas and/or object class to ensure that those numbers don't show up in the response
3. add at least one more `FinancialAccountsByProgramActivityObjectClass` with `tas1` and `object_class_1` and check the response to make sure the numbers summed correctly